src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -256,12 +256,12 @@ class GuiPluginList extends React.PureComponent<Props, State> {
       }
     }
 
-    const regionCode = { countryCode }
+    const isBuy = route.params.direction === 'buy'
     if (plugin.nativePlugin != null) {
-      await executePlugin({ guiPlugin: plugin, regionCode, paymentType, navigation, account })
+      await executePlugin({ isBuy, guiPlugin: plugin, regionCode: { countryCode }, paymentType, navigation, account })
```

Comment:
Keep with naming `direction` for consistency and make it an enum of 'buy' vs 'sell''. I believe @peachbits has a similar change in a PR that would conflict otherwise.